app/src/main/java/org/wikipedia/travel/landmarkpicker/LandmarkFragment.java

```diff
@@ -324,6 +352,16 @@ public void onClick(View v) {
             public void bindItem(LandmarkCard landmarkCard) {
                 textViewTitle.setText(landmarkCard.getTitle());
                 textViewDesc.setText(landmarkCard.getDesc());
+                checkBox.setChecked(landmarkCard.getChecked());
+                int semiTransparentWhite = Color.argb(ALPHA, RED, GREEN, BLUE);
+                //Apply a filter to the image for readability
+                landmarkImage.setColorFilter(semiTransparentWhite, PorterDuff.Mode.SRC_ATOP);
+                if (landmarkCard.getThumbUrl() != null) {
+                    landmarkImage.setVisibility(View.VISIBLE);
+                    landmarkImage.loadImage(Uri.parse(landmarkCard.getThumbUrl()));
+                } else {
+                    landmarkImage.setVisibility(View.GONE);
```

Comment:
If there's no thumbUrl, does the card remain blank?

Reply:
yes thats correct